.travis.yml

```diff
@@ -1,36 +1,30 @@
 language: python
+python:
+  - 2.7
+  - 3.5
+  - 3.6
+  - 3.7
+  - 3.8
+  - "nightly"
```

Comment:
I'd be keen not to test against the nightly build as this implies we support the nightly build, which I don't think we want.

Reply:
To me this is an early warning system, just testing for it we could find issues beforehand. Could be very very very specific in the README as per what's supported, while still knowing when things break in advance. Don't think the travis config should oblige us to anything.

But if this above doesn't make sense, will remove, as I am onboard with your logic - except for the premise. If my above premise is wrong, then your point stands completely.